index.html

```diff
@@ -34,7 +34,7 @@
       <span  class= "nav-humburger" id="humburger"><i class="fa humburger fa-bars"></i></span>
     </header>
 
-    <main class="main">
+    <div class="main">
```

Comment:
I really appreciate :heavy_check_mark:  the comment you made explaining the improvements to the student, but it could be better if you make a more friendly comment.  :smile: 
What do you think about including emojis to make your comment more human-friendly :alien: ?